dpctl/tests/test_service.py

```diff
@@ -151,13 +151,21 @@ def test_dev_utils():
             device.parent_device
 
 
-def test_syclinterface():
+@pytest.fixture
+def should_skip_syclinterface(request):
+    return request.config.getoption("--no-sycl-interface-test")
+
+
+def test_syclinterface(should_skip_syclinterface):
```

Comment:
May be something like `@pytest.mark.skipif` would be cleaner?
https://docs.pytest.org/en/stable/how-to/skipping.html#id1

Reply:
I think this change should be deferred to a later PR.

Essentially, it would replace `pass` with a call to `pytest.skip`.